gravity.sh

```diff
@@ -497,13 +498,28 @@ gravity_DownloadBlocklists() {
   # Use compression to reduce the amount of data that is transferred
   # between the Pi-hole and the ad list provider. Use this feature
   # only if it is supported by the locally available version of curl
-  if curl -V | grep -q "Features:.* libz"; then
+  curlVersion=$(curl -V)
+  if echo "${curlVersion}" | grep -q "Features:.* libz"; then
     compression="--compressed"
     echo -e "  ${INFO} Using libz compression\n"
   else
     compression=""
     echo -e "  ${INFO} Libz compression not available\n"
   fi
+
+  # Check if etag is supported by the locally available version of curl by
+  # comparing the version string being >= 7.68.0 (released Jan 2020)
+  # https://github.com/curl/curl/pull/4543 followed by
+  # https://github.com/curl/curl/pull/4678
+  if echo "${curlVersion}" | grep -q "curl 7\.[6-9][8-9]"; then
```

Comment:
Maybe this is a nicer way to compare
https://github.com/pi-hole/PADD/blob/9b94e4b7925b18e0cd8c1abc376ed1849676d90d/padd.sh#L1327-L1331

Reply:
Yeah, `awk` might be a better tool here instead of `grep`.